### PR TITLE
Adding Todo list project for codedamn

### PR DESCRIPTION
**Pull request by Misba9**
Merge my PR

### CODE REVIEW

File: todo-app/spec.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 name: Todo App
```
Comment (Misba9) [inline todo-app/spec.yml:1]: Add file

**Review by Misba9 — COMMENTED** [review]
Accept please

**Review by ayushete02 — APPROVED** [review]
Nice

**Comment by Pranavtechie** [comment]
Hey @Misba9, you should not change the complete preview of how an existing project looks like, we were supposed to write steps for an existing project or pick a new project of yourself.

**Pull request closed without merging**